Formula/gmssl.rb

```diff
@@ -6,6 +6,11 @@ class Gmssl < Formula
   license "Apache-2.0"
   head "https://github.com/guanzhi/GmSSL.git", branch: "master"
 
+  livecheck do
+    url :stable
+    strategy :github_latest
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

The `stable` URL in the formula is a tag archive (not a release asset) and we can successfully identify the latest version from the Git tags with a supplied regex, so the `GithubLatest` strategy isn't appropriate/necessary here.

Reply:
Thanks